src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -803,6 +803,12 @@ subroutine prepare_massflux_exports(IMPORT, EXPORT, PLE, dt, RC)
 ! OUTPUT PARAMETERS:
 !
       integer, optional, intent(out)  :: RC       ! Error code
+
+
+#ifdef ADJOINT
+      logical, save :: firstRun = .true.
```

Comment:
This is a local variable so should be further down in the subroutine with the other local variables.